src/main/java/com/aws/greengrass/mqttclient/MqttClient.java

```diff
@@ -239,6 +249,11 @@ protected MqttClient(DeviceConfiguration deviceConfiguration,
                     return;
                 }
 
+                if (node.childOf(DEVICE_MQTT_NAMESPACE)) {
+                    maxInFlightPublishes = Coerce.toInt(mqttTopics.lookup(MQTT_MAX_IN_FLIGHT_PUBLISHES_KEY));
+                    logger.atInfo().kv("value", maxInFlightPublishes).log("updating maxInFlightPublishes");
```

Comment:
this will unnecessarily log because you aren't checking that anything changed. I would not bother logging this.